.github/workflows/release.yml

```diff
@@ -54,7 +54,31 @@ env:
   NODE_OPTIONS: "--max-old-space-size=5120"
 
 jobs:
+  build:
+    timeout-minutes: 20
+    if: github.event.pull_request.merged == true
+    name: "Build Components"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+      - run: npm -v
+      - name: install
+        run: |
+          npm ci
+          nx run cli:bundle
+          nx run testing:bundle
+      - run: npx nx affected --target=lint $NX_CALCULATION_FLAGS
+      - run: nx affected:build $NX_CALCULATION_FLAGS --prod --exclude="demoshell"
+      - run: nx $NX_CALCULATION_FLAGS build demoshell --configuration production
+      - run: nx build demoshell --configuration production
```

Comment:
do we really need building demo shell for the release? it's not getting released from what I remember cc @eromano

Reply:
I guess you are right @DenysVuika

Reply:
ok so i'll find the minimum subset of commands needed to get libs needed for the release to be built